packages/stream/README.md

```diff
@@ -69,6 +69,50 @@ which promises settle.
 A stream is consequently a pair of queues that transport iteration results,
 one to send messages forward and another to receive acknowledgements.
 
+## Pump
```

Comment:
For each of the other sections of this README, a trivial code example helped tremendously. Could we have one for `pump` too? Especially if it fits with the others?